src/navigation/wallet/components/SendToAddress.tsx

```diff
@@ -115,6 +115,7 @@ const SendToAddress = () => {
     keys,
     id,
     currencyAbbreviation,
+    wallet.chain,
```

Comment:
You can use the extracted `chain` property from the wallet object (line 112)

Reply:
Oh, that's correct. Thanks!